README.adoc

```diff
@@ -23,13 +23,11 @@ permalink: /guides/jenkins-integration-with-kabanero/
 :page-guide-category: collections
 = Jenkins integration with Kabanero
 
-// *** For Alan's review ***
-
 Kabanero delivers a modern DevOps toolchain with pre-built Tekton pipelines. The Tekton open source project provides Kubernetes-style resources for declaring CI/CD-style pipelines. While it is advantageous to run both CI and CD with Tekton to leverage the power of Kubernetes, some might want to reuse Jenkins assets or skills.
 
 With Kabanero, you can customize the deployment section of the pipeline to use Jenkins.
 
-In this guide, you will learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image.
+In this guide, you will learn how to use Tekton to build an image and how to use Jenkins to deploy an application that uses that image. Tekton pipelines are created manually in this guide. A separate guide will address how to use automated webhooks to create and run Tekton pipelines.
```

Comment:
Should we update
"Tekton pipelines are created manually in this guide. A separate guide will address how to use automated webhooks to create and run Tekton pipelines."
To
"In this guide, Tekton pipelines are created manually. A future guide enhancement will address how to use automated webhooks to create and run Tekton pipelines."

- I think we discussed keeping this all in one guide rather than creating a new separate guide. If we use more generic lanugage, we can make that call when we start the rewrite.

Reply:
I like charlotte's suggestions.   Thank you!

Reply:
LGTM.
Also, pls change OpenShift4.2 to OKD3.11 for now.